### ISSUE TITLE
Scroll to given row and section

### ISSUE THREAD

**Issue by MrJai** [dharmendraiosdev/flutter_section_list_view#3]
There is a native function in iOS for [tableView scrollToRowAtIndexPath:IndexPath]. Is it possible to implement such functionality in this package?

**Comment by dharmendraiosdev** [comment]
Yes, @MrJai I think, it should be possible, and thanks for bringing this up.
I will look into it and try to include it in my next update soon, for sure.

**Comment by MrJai** [comment]
Thank you @dharmendraiosdev. I achieved it using FlutterTableView and AutoScrollTag around each ListItem. For the keys, I used the same index as the index of the cell.
